namerd/iface/control-http/src/main/scala/io/buoyant/namerd/iface/HttpControlService.scala

```diff
@@ -341,15 +341,28 @@ class HttpControlService(storage: DtabStore, delegate: Ns => NameInterpreter, na
   private[this] def getBind(ns: String, path: Path, extraDtab: Option[String]): Activity[NameTree[Name.Bound]] =
     extraDtab match {
       case Some(dtab) => delegate(ns).bind(Dtab.read(dtab), path)
-      case _ => bindingCache.get((ns, path))
+      case _ => bindingCache.get((ns, path))._1
     }
 
+  type NsPath = (String, Path)
```

Comment:
what about case classes instead of type aliases for both of these?  that way the fields can be referred to by name instead of by `._1`